src/libravi/RaviSignalingConnection.ts

```diff
@@ -434,8 +455,22 @@ class RaviSignalingWebSocketImplementation {
         signalingConnection._handleStateChange({}, RaviSignalingStates.CLOSED);
```

Comment:
Is there any chance to get here? If so, don't we want all the same things to happen, in terms of setting _webSocket to null and removing event listeners?

Reply:
Yup, agree -- I think this _should_ just be `if (! this._websocket) return etc.` (to catch the "this._websocket is already null" case) and let the `if (this._webSocket.readyState === crossPlatformWebSocket.CLOSED)` fall through to the rest of the stuff that this method does.